lib/fast_gettext/translation_repository/yaml.rb

```diff
@@ -44,7 +44,8 @@ def find_and_store_files(options)
           # that is, we suppose it to be named `qq.yml` or `foo.qq.yml` where
           # `qq` stands for a locale name
           locale = File.basename(yaml_file, '.yml').split('.').last
-          @files[locale] = load_yaml(yaml_file, locale)
+          @files[locale] ||= {}
+          @files[locale].merge! load_yaml(yaml_file, locale)
```

Comment:
could turn that into a single line with `(@files[locale] ||= {}).merge!`

Reply:
:heavy_check_mark: